proto/babylon/incentive/rewards.proto

```diff
@@ -5,7 +5,7 @@ import "gogoproto/gogo.proto";
 import "cosmos/base/v1beta1/coin.proto";
 import "cosmos_proto/cosmos.proto";
 
-option go_package = "github.com/babylonlabs-io/babylon/x/incentive/types";
+option go_package = "github.com/babylonlabs-io/babylon/v2/x/incentive/types";
```

Comment:
Oh I see some protos have this changed to v2, some not. Should we have it changed in all of them ? (if so maybe we can do that in separate pr)